libs/langgraph/langgraph/pregel/loop.py

```diff
@@ -480,6 +484,15 @@ def tick(
             self.status = "out_of_steps"
             return False
 
+        # If updated channels is available, we project only a subset of the nodes.
+        # since only those nodes have been updated?
+        triggered_nodes: set[str] = set()
```

Comment:
I think prepare_next_tasks should receive updated_nodes, and do this work inside?